rpcs3/Emu/Cell/Modules/cellSsl.cpp

```diff
@@ -23,14 +25,22 @@ s32 cellSslEnd()
 	return CELL_OK;
 }
 
-std::string getCert(const std::string certPath, const int certID)
+std::string getCert(const std::string certPath, const int certID, const bool isNormalCert)
 {
-	//Stupid cellssl
 	int newID = certID;
-	if (certID == 6)
-		newID = 23;
-	if (certID > 6 && certID < 23)
-		newID = certID - 1;
+
+	// The 'normal' certs have some special rules for loading.
+	if (isNormalCert && certID >= BaltimoreCert && certID <= GTECyberTrustGlobalCert)
+	{
+		if (certID == BaltimoreCert)
+			newID = GTECyberTrustGlobalCert;
+		else if (certID == Class3G2V2Cert)
+			newID = BaltimoreCert;
+		else if (certID == EntrustNetCert)
+			newID = ClassSSV4Cert;
+		else
+			newID = certID - 1;
```

Comment:
newID == certID in this moment why not make it "--newID"? It should be faster than subtraction and slihghtly faster than "newID--"

Reply:
It won't be faster and it want matter in this case anyway.

Reply:
Ok.